src/ui/public/indices/get_indices.js

```diff
@@ -1,6 +1,10 @@
 import { pluck, reduce, size } from 'lodash';
 
-const getIndicesFromResponse = json => {
+const getIndexNamesFromAliasesResponse = json => {
+  if (json.status === 404) {
```

Comment:
I was thinking about this too... after chewing it over, I think we should consider it incidental that it's handled on line 30. My basis for this reasoning is that this function doesn't "know" about line 30. So by handling the 404 case we've future-proofed it. We'll never need to change it even if the surrounding code changes (e.g. by exporting this function and using it elsewhere, or by removing the condition on line 30). What do you think?